WORKSPACE

```diff
@@ -120,16 +120,6 @@ install_pypiserver()
 
 pip_parse(
     name = "dev_pip",
-    experimental_requirement_cycles = {
```

Comment:
Is something else handling these cycles, or did they go away?

Reply:
They got removed upstream.